finagle-http/src/main/scala/com/twitter/finagle/http/filter/StatsFilter.scala

```diff
@@ -1,9 +1,29 @@
 package com.twitter.finagle.http.filter
 
-import com.twitter.finagle.{Service, SimpleFilter}
-import com.twitter.finagle.stats.StatsReceiver
+import com.twitter.finagle._
 import com.twitter.finagle.http.{Request, Response, Status}
-import com.twitter.util.{Duration, Future, Return, Stopwatch, Throw}
+import com.twitter.finagle.stats.StatsReceiver
+import com.twitter.util._
+
+object StatsFilter {
+  val role = Stack.Role("HttpStatsFilter")
+
+  def module[Req <: Request]: Stackable[ServiceFactory[Req, Response]] =
```

Comment:
I've put the type-param in there because the filter itself has it as well and limiting it in the module seemed strange. Additionally, our applications use types that inherit from `RequestProxy` so we actually like the presence of the type-param in the filter and think the api would be less rich and flexible without it.